libgadget/treewalk.c

```diff
@@ -379,6 +406,10 @@ static int ev_primary(TreeWalk * tw)
         tw->Nexport --;
     }
 
+    if(tw->BufferFullFlag) {
+        message(1, "Tree export buffer full with %d particles. Increase BufferSize if possible.\n", tw->Nexport);
```

Comment:
"This is not fatal but makes treewalk slower."

Reply:
Done.

Reply:
Ideally we would increase the BufferSize dynamically at this point. But as I never hit this condition in practice I am unsure how exactly we would resize it.